Documentation/cluster-crd.md

```diff
@@ -177,6 +177,43 @@ spec:
       deviceFilter: "^sd."
 ```
 
+### Storage Configuration: Cluster wide Directories
+
+This example is based up on the [Storage Configuration: Specific devices](#storage-configuration-specific-devices).
+Individual nodes can override the cluster wiede specified directories list.
```

Comment:
typo: "wiede".  that looks like a German word :)

Reply:
`wieder` would be German, should have been `wide` ;)